apps/k8s-io/deployment.yaml

```diff
@@ -34,7 +34,7 @@ spec:
           # Map all keys to files.
       containers:
       - name: nginx
-        image: nginx:1.25-alpine
+        image: nginx:1.26-alpine
```

Comment:
can we please at least pin the digest?

this is a bit terrifying in the critical path to e.g. binary downloads

Reply:
Updated. PTAL.